components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/internal/IdentityEventServiceComponent.java

```diff
@@ -148,19 +147,5 @@ protected void setIdentityCoreInitializedEventService(IdentityCoreInitializedEve
     /* reference IdentityCoreInitializedEvent service to guarantee that this component will wait until identity core
          is started */
     }
-
```

Comment:
why did we remove this? 

This was to ,

 service to guarantee that this component will wait until identity core is started

Reply:
That is not removed. remove idp.mgt componet which is not used and it cause cyclic dependency